hardware/Schematic-Design-Guide.md

```diff
@@ -26,42 +33,58 @@ There are relative reference designs for all of the solutions in the website. Ba
 
   32/48/68 pins QFN package and 125pins BGA package for EFR32 chips.
```

Comment:
We also have QFN40 which should be included

Reply:
Corrected.